moveit_setup_assistant/src/widgets/start_screen_widget.cpp

```diff
@@ -305,6 +305,15 @@ void StartScreenWidget::loadFilesClick()
   }
   else
   {
+    if ( !create_new_package_ && config_data_->author_name_.empty() && config_data_->author_email_.empty() )
+    {
+      QMessageBox::warning( this, "Loading Project Without Author Information",
```

Comment:
Consider renaming the window title to "Missing author information". The proposed one seems quite long for a window title.